src/Command/CommandBase.php

```diff
@@ -1089,7 +1089,8 @@ public static function getAliasCache() {
    */
   protected function doGetApplicationFromAlias($application_alias) {
     $acquia_cloud_client = $this->cloudApiClientService->getClient();
-    $acquia_cloud_client->addQuery('filter', 'hosting=@*' . $application_alias);
+    $filter_value = self::createHostingIdParamValue($application_alias);
+    $acquia_cloud_client->addQuery('filter', 'hosting=' . $filter_value);
```

Comment:
```suggestion
    $acquia_cloud_client->addQuery('filter', 'hosting=@*:' . $filter_value);
```

Just changing this (and nothing else) seems to fix the issue for me.